cmd/explaintest/r/explain_complex.result

```diff
@@ -156,9 +156,9 @@ Projection_10	0.00	root	test.dt.id, test.dt.aid, test.dt.pt, test.dt.dic, test.d
     ├─TableReader_41	0.00	root	data:Selection_40
     │ └─Selection_40	0.00	cop	eq(test.dt.bm, 0), eq(test.dt.pt, "ios"), gt(test.dt.t, 1478185592), not(isnull(test.dt.dic))
     │   └─TableScan_39	10000.00	cop	table:dt, range:[0,+inf], keep order:false, stats:pseudo
-    └─IndexLookUp_18	3.33	root	
+    └─IndexLookUp_18	0.00	root	
       ├─IndexScan_15	10.00	cop	table:rr, index:aid, dic, range: decided by [eq(test.rr.aid, test.dt.aid) eq(test.rr.dic, test.dt.dic)], keep order:false, stats:pseudo
-      └─Selection_17	3.33	cop	eq(test.rr.pt, "ios"), gt(test.rr.t, 1478185592)
+      └─Selection_17	0.00	cop	eq(test.rr.pt, "ios"), gt(test.rr.t, 1478185592)
```

Comment:
Note: the real row count here is `0.003333333333333334`, which is reasonable, since we are using `count := string(strconv.AppendFloat([]byte{}, p.statsInfo().RowCount, 'f', 2, 64))` in `prepareOperatorInfo`, it is displayed as `0.00`.